ports/stm32f4/mpconfigport.mk

```diff
@@ -21,6 +21,10 @@ CIRCUITPY_ANALOGIO = 1
 CIRCUITPY_MICROCONTROLLER = 1
 CIRCUITPY_BUSIO = 1
 CIRCUITPY_OS = 1
+CIRCUITPY_STORAGE = 1
+CIRCUITPY_RANDOM = 1
+CIRCUITPY_USB_HID = 1
+CIRCUITPY_USB_MIDI = 1
```

Comment:
Aren't these already on?

Reply:
These are the ones that didn't make it in with the PR due to reliance on stub presence. I'll re-add with the `template-port` if we decide to swing back to that.